features/support/env.rb

```diff
@@ -8,6 +8,7 @@
 require 'simplecov'
 require 'cucumber/timecop'
 require 'capybara/poltergeist'
+require 'cucumber/rspec/doubles'
```

Comment:
I knew I've seen this somewhere. Can you remove the same require from here?

https://github.com/otwcode/otwarchive/blob/df6ce85db5cb1a857186215e43cd0cafba3234a2/features/step_definitions/search_steps.rb#L1